lib/src/main/kotlin/com/swmansion/starknet/account/StandardAccount.kt

```diff
@@ -23,6 +24,7 @@ class StandardAccount(
     private val provider: Provider,
 ) : Account {
     private val version = Felt.ONE
+    private val estimateVersion: BigInteger = BigInteger.valueOf(2).pow(128).add(BigInteger(version.decString()))
```

Comment:
```suggestion
    private val estimateVersion: BigInteger = BigInteger.valueOf(2).pow(128).add(version.value)
```

I think you can use the underlying BigInteger directly here.